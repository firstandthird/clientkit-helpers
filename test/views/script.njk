{{ 'test.js' | script | safe }}
{{ 'test2.js' | script(true) | safe }}
