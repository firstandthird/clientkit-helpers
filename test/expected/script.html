<script src="http://localhost/test.js" crossorigin="anonymous"></script>
<script src="http://localhost/test2.js" crossorigin="anonymous" defer></script>
